src/main/java/ca/mcgill/mcb/pcingola/snpEffect/commandLine/SnpEffCmdEff.java

```diff
@@ -279,11 +284,16 @@ public void annotate(VcfEntry vcfEntry) {
 	@Override
 	public boolean annotateFinish() {
 		boolean ok = true;
-		if (createSummary && (summaryFile != null)) {
+		if (createSummary) {
 			// Creates a summary output file
-			if (verbose) Timer.showStdErr("Creating summary file: " + summaryFile);
-			if (createCsvSummary) ok &= summary(SUMMARY_CSV_TEMPLATE, summaryFile, true);
-			else ok &= summary(SUMMARY_TEMPLATE, summaryFile, false);
+			if (verbose) {
+				if (createSummaryCSV && (summaryFileCSV != null))
+					Timer.showStdErr("Creating summary file: " + summaryFileCSV);
+				if (createSummaryHTML && (summaryFileHTML != null))
+					Timer.showStdErr("Creating summary file: " + summaryFileHTML);
+			}
+			if (createSummaryCSV) ok &= summary(SUMMARY_CSV_TEMPLATE, summaryFileCSV, true);
```

Comment:
You can collapse this into the previous if statement, they are checking the same.